### PR TITLE
docs(): add Nicolas Frizzarin to GDE resources

### PR DESCRIPTION
**Pull request by Nicoss54**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -779,6 +779,14 @@
...
     "groups": ["GDE"]
   },
+  "nicoss54": {
+    "name": "Nicolas Frizzarin",
+    "picture": "nicolasfrizzarin.jpeg",
+    "twitter": "Nicooss54",
+    "website": "https://dev.to/nicoss54",
+    "bio": "Google Developer Expert in Web Technologies and Angular and OpenJs Foundation Member, Nicolas is a FontEnd specialist and user Interface Designer from Metz, France. He works as a Developer Advocate and Senior Staff Engineer at Sfeir Luxembourg. His daily mission is to help teams to make robust, powerful and beautiful web applications Besides that, Nicolas is a speaker, and likes to contribute to the community by giving talks, trainings, writing articles or participating in open source projects",
```
Comment (MarkTechson) [inline aio/content/marketing/contributors.json:787]: There is some punctuation missing. a period should be added after "beautiful web applications." and at the end after "open source projects."

Like this:
"and beautiful web applications. Besides that, Nicolas is a speaker, and likes to contribute to the community by giving talks, trainings, writing articles or participating in open source projects."

**Review by MarkTechson — CHANGES_REQUESTED** [review]
couple small changes. but LGTM after that.

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (Nicoss54) [reply aio/content/marketing/contributors.json]: I think it's done, thanks a lot for the review. I also rebase the branch into the main

**Comment by JeanMeche** [comment]
@Nicoss54, no need for the braces when there is no scope. This is why the Linting job is failing !

**Review by MarkTechson — APPROVED** [review]
LGTM

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit f3eebaf76ce30e0c7c1668f695247f92e590990e.

**Pull request closed without merging**